SConstruct

```diff
@@ -3,11 +3,13 @@ import subprocess
 import sysconfig
 import platform
 import numpy as np
+from pathlib import Path
 
 arch = subprocess.check_output(["uname", "-m"], encoding='utf8').rstrip()
 if platform.system() == "Darwin":
   arch = "Darwin"
 
+os.environ['PYTHONPATH'] = str(Path(sysconfig.get_paths()['data']).parent)
```

Comment:
this isn't the right fix. this should already be set before you get to this point